Car.h

```diff
@@ -1,9 +1,12 @@
 #pragma once
 
+#include <string>
+#include "Vehicle.h"
+
 struct Car : public Vehicle
```

Comment:
Don't include what you don't use.  you aren't using `<string>` in this header file
`struct` is `public` by default.  remove this keyword
- [ ] revise